src/poliastro/core/threebody/cr3bp_quantities_calculations.py

```diff
@@ -0,0 +1,48 @@
+"""
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+"""
+
+from astropy import units as u
+
+
+def calculate_mu(mu1, mu2):
+    """Calculate mu of CR3BP
+
+    Parameters
+    ----------
+    mu1: float, km^3*s^-2 or m^3*s^-2
+        mu of P1
+    mu1: float, km^3*s^-2 or m^3*s^-2
+        mu of P2
+
+    Returns
+    -------
+    mu: float, dimensionless
+        mu2/(mu1+mu2), mu2<mu1
+    """
+    return (mu2 / (mu1 + mu2)) * u.one
+
+
+def calculate_tstar(mu1, mu2, lstar):
+    """Calculate t* of CR3BP
```

Comment:
```suggestion
    r"""Calculate t* of CR3BP
```

This will fix the pre-commit error